kylin-spark-project/kylin-spark-common/src/main/scala/org/apache/spark/sql/execution/datasource/FilePruner.scala

```diff
@@ -78,6 +77,8 @@ class FilePruner(cubeInstance: CubeInstance,
                  val options: Map[String, String])
   extends FileIndex with ResetShufflePartition with Logging {
 
+  val MAX_SHARDING_SIZE_PER_TASK: Long = cubeInstance.getConfig.getMaxShardingSizeMBPerTask * 1024 * 1024
```

Comment:
this line will exceed 100 chars, please change to:
```
  val MAX_SHARDING_SIZE_PER_TASK: Long =
    cubeInstance.getConfig.getMaxShardingSizeMBPerTask * 1024 * 1024
```

Reply:
done